plugins/wasmembed/secure-enclave.cpp

```diff
@@ -538,20 +517,23 @@ class SecureFunction : public CInterfaceOf<IEmbedFunctionContext>
     {
         TRACE("WASM SE getStringResult %zu", wasmResults.size());
         auto ptr = wasmResults[0].i32();
-        auto data = wasmEngine->getData(wasmName);
+        auto data = wasmStore->getData(wasmName);
         uint32_t strPtr;
+        std::string encoding;
         uint32_t bytes;
-        std::tie(strPtr, bytes) = load_string(data, ptr);
-        rtlUtf8ToStrX(chars, result, bytes, reinterpret_cast<const char *>(&data[strPtr]));
+        std::tie(strPtr, encoding, bytes) = load_string(data, ptr);
+        size32_t codepoints = rtlUtf8Length(bytes, &data[strPtr]);
```

Comment:
Should encoding be being used?

Reply:
Not currently as everything is always a utf8 - this is more to keep in sync with official ABI and in the future when other encodings are supported.